### PR TITLE
Added documentation for Inline Variant of Radio button and Checkbox

### PR DESCRIPTION
**Pull request by methodphoto**

### CODE REVIEW

File: docs/components/ns-inputter.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ description: Molecule | Inputter component.
 
 import { StorybookStory } from '../../includes/storybook-story.js'
 import { Tokens } from '../../includes/tokens.js'
-import { ComponentPlacement } from '../../includes/component-placement.js'
+import { ComponentPlacement } from '../../includes/component-placement.js'![image](https://user-images.githubusercontent.com/5929591/186429419-0a53e226-41ba-4e86-9f51-988570800a8b.jpeg)
```
Comment (andij) [inline docs/components/ns-inputter.md:7]: Looks like a rogue image was committed.

```suggestion
import { ComponentPlacement } from '../../includes/component-placement.js'
```

File: docs/components/ns-inputter.md [hunk 2]
```diff
@@ -85,6 +86,30 @@ To match this heading's size with other `ns-inputter` labels', group all associa
 | Allow user to opt in as opposed to opt out |  |
 | Consider splitting question out for when more than 5 checkboxes are required |  |
 
+### Inine Variant of Radio buttons and Checkboxes
```
Comment (PtitBen) [inline docs/components/ns-inputter.md:89]:
```suggestion
### Inline Variant of Radio buttons and Checkboxes
```

File: docs/components/ns-inputter.md [hunk 3]
```diff
@@ -85,6 +86,30 @@ To match this heading's size with other `ns-inputter` labels', group all associa
...
+
+This is aimed at a small cluster of contextual questions with a few short answers. It works well especially for mobile as there is a larger hit area for the customer as well as there being a manageable cognitive load. The decoration allows the customer to focus on the answers.
+
+#### When to use it
+
+When you are trying to get a simple set of answers from the customer where the answers are short, and the number of potential answers is below 4-5.
+
+Use the contextual helper text to explain a more complicated question or to offer help or context. If the question and answer is really obvious then you may not need a contextual helper.
```
Comment (PtitBen) [inline docs/components/ns-inputter.md:97]:
```suggestion
Use the contextual [helper text](https://nucleus.design/docs/components/ns-inputter#helper-text) to explain a more complicated question or to offer help or context. If the question and answer is really obvious then you may not need a contextual helper.
```

Maybe add a link to the helper text documentation

**Review by PtitBen — COMMENTED** [review]
Only a couple things, thanks for the contribution Richard !

**Review by methodphoto — COMMENTED** [review]
These updates are all good!

**Review by PtitBen — APPROVED** [review]

**Pull request merged**